tyssue/core/objects.py

```diff
@@ -142,6 +146,11 @@ def edge_df(self, value):
 
     @property
     def cell_df(self):
+        #- BC -#
+        # Recommends to test if the epithelium has a 'cell' df before returning,
+        # some epitheliums don't require this object (2D)
+        # Should it raise a warning/error if the user asks for
+        # a cell_df from an eptm that doesn't have one ?
```

Comment:
There will be a `KeyError` risen due to the absence of the data in the `self.datasets` dictionary. But it might be clearer if we raise a more explicit error before hand.